pages/recipe/[recipeSlug]/[id].tsx

```diff
@@ -0,0 +1,108 @@
+import { GetStaticPaths, GetStaticProps } from "next"
+import Card from "react-bootstrap/Card"
+import { getHostname, stringToSlug } from "@/utilities"
+
+export type Props = {
+  created_at: string
+  edited_at: string
+  id: string
+  ingredients: {
+    abstract_ingredient_id: string
+    id: string
+    name: string
+    quantity: number
+    recipe_id: string
+    units: string
+  }[]
+  instructions: string
+  name: string
+  servings: number
+  source: string
+}
+
+const Recipe = (props: Props) => {
+  const { ingredients, instructions, name, servings, source } = props
+  const url = new URL(source)
+  return (
+    <Card>
+      <Card.Header>
+        <Card.Title>{name}</Card.Title>
+        <Card.Link
+          href={source}
+          target="_blank"
+          style={{ display: "block" }}
+        >
+          {url.hostname}
+          <svg
```

Comment:
Nit: can we wrap this SVG in a helper function so (1) it doesn't clutter up the Card component and (2) we can potentially use it other places?